src/Actions/Actions/ScreenShotAction.cs

```diff
@@ -16,6 +16,10 @@ namespace Axe.Windows.Actions
     [InteractionLevel(UxInteractionLevel.NoUxInteraction)]
     public static class ScreenShotAction
     {
+        // unit test hooks
+        internal static Func<DataManager> GetDataManager = () => DataManager.GetDefaultInstance();
```

Comment:
Code coverage shows that the call to GetDefaultInstance is never being called here. That would suggest that this line can be removed